source/blog/embertimes-template.md

```diff
@@ -79,7 +79,7 @@ responsive: true
 
 Want to write for the Ember Times? Have a suggestion for next week's issue? Join us at #support-ember-times on the [Ember Community Discord](https://discordapp.com/invite/zT3asNS) or ping us [@embertimes](https://twitter.com/embertimes) on Twitter. 
```

Comment:
We could make this section #embertimes to match last week's Goodbits, or no heading

Reply:
Yes, the #embertimes title header looks cool, I'd be in for that!

Reply:
ok changed it, let's give it a try